ci/do_ci.sh

```diff
@@ -407,6 +407,7 @@ elif [[ "$CI_TARGET" == "fix_format" ]]; then
 
   echo "fix_format..."
   "${ENVOY_SRCDIR}"/tools/code_format/check_format.py fix
+  "${ENVOY_SRCDIR}"/tools/code_format/format_python_tools.sh fix
```

Comment:
these do not want to be added back - there is a separate tool for doing this - see `tools/code_format/check_python.py` - dont add here tho